src/ensembl/src/shared/components/layout/DocLayout.scss

```diff
@@ -1,56 +1,86 @@
 @import 'src/styles/common';
 
-$sidebarContentWidth: 320px;
+$topbarHeight: 48px;
+$globalNavWidth: 320px;
+$localNavWidth: 320px;
+$columnPadding: 36px;
 
 .docLayout {
   height: 100%;
   min-height: 550px; // based on the current interface of genome browser
   overflow: hidden;
+  display: grid;
+  grid-template-columns: $globalNavWidth auto $localNavWidth;
+  grid-template-rows: $topbarHeight auto;
```

Comment:
we can use the grid-template-areas here:

```
grid-template-areas: "app-name search-box search-box"
                                    "global-nav main local-nav"
```

and then, in the corresponding child classes:

```
.appHeader {
  grid-area: "app-name";
}

.searchBox {
  grid-area: "search-box";
}
```

that might be easier to read and doesn't even need the grid comment, because the way `grid-template-areas` is written resembles ascii art. 

This is entirely optional; but see if you like this way of describing the grid better.

Reply:
I don't really like the grid-area syntax - the use of repeating area names is not very intuitive or visual IMHO.